ios/Classes/NetIamyellowTiwsWSProxy.m

```diff
@@ -121,8 +124,9 @@ -(void)close:(id)args
 
 -(void)send:(id)msg
 {
-    ENSURE_SINGLE_ARG(msg, NSString);
-
+    // not sure why tianium is passing a __NSArrayM with inside the TiBLob
+    if (![msg isKindOfClass:[NSString class]])
+        msg = [msg[0] data];
```

Comment:
Philosophically JavaScript functions accepts only two different parameters: `thisValue` and `args`. In the Obj-C Kroll this is respected and everything comes to native functions as a list of arguments. That’s the reason behind the use of `ENSURE_SINGLE_ARG` macro pretty everywhere.